packages/models/src/lib/category-config.ts

```diff
@@ -13,22 +13,26 @@ type _RefsList = {
   plugin?: string;
 }[];
 
+const categoryRefSchema = weightedRefSchema(
+  'Weighted references to audits and/or groups for the category',
+  'Slug of an audit or group (depending on `type`)',
+).merge(
+  z.object({
+    type: z.enum(['audit', 'group'], {
+      description:
+        'Discriminant for reference kind, affects where `slug` is looked up',
+    }),
+    plugin: slugSchema(
+      'Plugin slug (plugin should contain referenced audit or group)',
+    ),
+  }),
+);
+
+export type CategoryRefSchema = z.infer<typeof categoryRefSchema>;
```

Comment:
The type name shouldn't have the `Schema`-suffix. The naming convention is `const fooSchema = ...` and `type Foo = z.infer<typeof fooSchema>`.

```suggestion
export type CategoryRef = z.infer<typeof categoryRefSchema>;
```

Reply:
done